deploy/002_deploy_optimism_adapter.ts

```diff
@@ -18,6 +18,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       L1_ADDRESS_MAP[chainId].weth,
       L1_ADDRESS_MAP[chainId].optimismCrossDomainMessenger,
       L1_ADDRESS_MAP[chainId].optimismStandardBridge,
+      L1_ADDRESS_MAP[chainId].l1UsdcAddress,
```

Comment:
nit: suggest commenting out the `cctpTokenMessenger` line and adding in a zeroAddress for production so we don't accidentally deploy this with full cctp support yet

This comment applies to all deploy scripts as well

Reply:
Good call - per the sync today I've set all spoke/adapters `cctpTokenMessenger` contract to `ZERO_ADDRESS`.